sp/src/game/server/hl2/npc_antlion.cpp

```diff
@@ -3206,8 +3206,13 @@ bool CNPC_Antlion::HandleInteraction( int interactionType, void *data, CBaseComb
 		Flip();
 
 		// If the Combine soldier is a player, don't deal damage on top of flipping
+		// No, at the beginning of Ashes the non-damaging melee would be a disappointment like in Doom Eternal - HEVcrab
```

Comment:
Please fix this comment. There's no need for the back and forth here, just change the original comment to explain the behavior.

Reply:
> Please fix this comment. There's no need for the back and forth here, just change the original comment to explain the behavior.

I will, thank you. BTW, maybe 5 kicks are too much? 3 kicks will match 10 damage per kick like in the skill config